templates/patterns/timeline/timeline.ui_patterns.yml

```diff
@@ -0,0 +1,41 @@
+timeline:
+  label: "Timeline"
+  description: "A timeline is a way of displaying a list of events in chronological order."
+  fields:
+    limit:
+      type: "numeric"
+      label: "Limit"
+      description: "Number of items to show"
+      preview: 3
+      escape: false
```

Comment:
Can't find for what this element
https://ui-patterns.readthedocs.io/en/8.x-1.x/content/patterns-definition.html
Is this have any value?

Reply:
Yes, otherwise it will be escaped and it will be used as string not int.